lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -1363,6 +1367,11 @@ def test_view_certificate_link(self):
         certificates[0]['is_active'] = False
         self.store.update_item(self.course, self.user.id)
 
+        # Re-enable certificate self-generation for this course because whenever
+        # course content changes then self-generation changes according to pacing
+        # (EDUCATOR-394)
+        certs_api.set_cert_generation_enabled(self.course.id, True)
```

Comment:
We can make but as the course is instructor-paced so i thought we should modify the test according to it.